project/ui/qprof-ui.ipynb

```diff
@@ -594,22 +594,48 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "a8232375-7334-48eb-b691-c43312964439",
-   "metadata": {},
+   "id": "efeb285b-60de-4474-bdd3-f4f1998ace2b",
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": [
-    "accordion = widgets.Accordion(children=[manual_query, output_initial, advanced_display_settings_widget], titles=('Manual Queries','Execution/Error Details','Advanced Output Settings'))\n",
-    "logging.info(f'[Query Profile Tree Page] Trying to display the error log at the bottom of the page')\n",
-    "accordion\n"
+    "button_run_explain_plan = widgets.Button(description=\"Run Explain Plan\")\n",
+    "explain_output = widgets.Output()\n",
+    "\n",
+    "def on_button_clicked_run_explain_plan(b):\n",
+    "    with explain_output:\n",
+    "        explain = qprof.get_qplan_explain(display_trees= False)\n",
+    "        for i in range(len(explain)):\n",
+    "            to_print = explain[i]\n",
+    "            if type(to_print) is str:\n",
+    "                print(to_print)\n",
+    "\n",
+    "\n",
+    "def on_button_clicked_run_query(b):\n",
+    "    output_run_query.clear_output(wait=True)\n",
+    "    global query_val, target_schema_val_2, key_val_2\n",
+    "    with output_run_query:\n",
+    "        query_val = query.value\n",
+    "        logging.info(f\"\"\"[Query Profile Tree Page] Running the following query manually :{query_val}.\"\"\")\n",
```

Comment:
I don't get this log message. Does manually mean the users are going to run the query themselves?

Reply:
I updated the language to make it clearer that the "custom query" is being run.